dev/RadialGradientBrush/RadialGradientBrush.idl

```diff
@@ -0,0 +1,37 @@
+﻿namespace MU_XM_NAMESPACE
+{
+
+[WUXC_VERSION_PREVIEW]
+[webhosthidden]
+[MUX_PROPERTY_CHANGED_CALLBACK(TRUE)]
+[MUX_PROPERTY_CHANGED_CALLBACK_METHODNAME("OnPropertyChanged")]
```

Comment:
We are prefering individualized property changed callbacks via the [MUX_PROPERTY_CHANGED_CALLBACK(TRUE)] tag for individual properties. See ProgressBar.idl for an example

Reply:
converted to per-property callbacks